Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -887,8 +908,13 @@ const TextInput = createReactClass({
         'TextInput children are not supported on Windows.'
     );
 
-    let textContainer;
-    if (props.secureTextEntry) {
+    const tabIndex = this.props.tabIndex || 0;
+    const windowsTabFocusable = this.props.editable && tabIndex >= 0;
+
+    var textContainer;
+    if (this.props.secureTextEntry) {
+      // Note: PasswordBoxWindows is not the native component, but a JS one fronting that native one.
```

Comment:
>Note [](start = 9, length = 4)

Should we open a GitHub issue for this and reference it here?

Reply:
[https://github.com/Microsoft/react-native-windows/issues/1569](https://github.com/Microsoft/react-native-windows/issues/1569)

---
In reply to: [157099888](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r157099888) [](ancestors = 157099888)